### PR TITLE
Diffuse Probe Grid component docs

### PR DESCRIPTION
**Pull request by chanmosq**
Updated the Diffuse Probe Grid component's reference page.

### CODE REVIEW

File: content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md [hunk 1]
```diff
@@ -5,18 +5,61 @@ description: "Open 3D Engine (O3DE) Diffuse Probe Grid component reference."
 toc: true
 ---
 
-{{< preview-new >}}
+The **Diffuse Probe Grid component** creates a volume of light probes that provide diffuse global illumination (GI) within the specified area. Each probe in the volume uses real-time ray tracing to capture the *irradiance*, or surrounding diffuse light environment. Real-time ray tracing casts several hundred rays in different directions around each probe. The probe stores lighting information at each point of intersection between the ray and the mesh. Then, it creates an irradiance texture out of the ray traced data and applies the texture to each mesh.
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md:8]:
```suggestion
The **Diffuse Probe Grid** component creates a volume of light probes that provide diffuse global illumination (GI) within the specified area. Each probe in the volume uses real-time ray tracing to capture the *irradiance*, or surrounding diffuse light environment. Real-time ray tracing casts several hundred rays in different directions around each probe. The probe stores lighting information at each point of intersection between the ray and the mesh. Then, it creates an irradiance texture out of the ray traced data and applies the texture to each mesh.
```

File: content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md [hunk 2]
```diff
@@ -5,18 +5,61 @@ description: "Open 3D Engine (O3DE) Diffuse Probe Grid component reference."
...
+
+### Bake Textures
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Bake Textures** | Bakes the surrounding diffuse light environment to a texture that'll be used when **Editor Mode** or **Runtime Mode** is set to `Baked`. You can only bake textures when Editor Mode or Runtime Mode is set to `Real Time (Ray-Traced)`. | - | - |
+
+### Probe Spacing
+
+Probe spacing is the distance between probes (in meters) along each axis within the volume. Less spacing results in more probes, which increases the fidelity of the GI at the expense of performance and memory. The maximum extents of the space depends on the **Box Shape component's** dimension.
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md:31]:
```suggestion
 Probe spacing is the distance between probes (in meters) along each axis within the volume. Less spacing results in more probes, which increases the fidelity of the GI at the expense of performance and memory. The maximum extents of the space depends on the **Box Shape** component's dimension.
```

**Review by micronAMZN — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md [hunk 3]
```diff
@@ -5,18 +5,61 @@ description: "Open 3D Engine (O3DE) Diffuse Probe Grid component reference."
...
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **X-Axis** | The amount of space between probes along the x-axis. The space must be within the Box Shape component's dimension along the x-axis. | `0.0` to the Box Shape component's `Dimensions`-`X` property. | The lowest value from the Box Shape component's `Dimensions`-`XYZ` property. |
+| **Y-Axis** | The amount of space between probes along the y-axis. The space must be within the Box Shape component's dimension along the y-axis. | `0.0` to the Box Shape component's `Dimensions`-`Y` property. | The lowest value from the Box Shape component's `Dimensions`-`XYZ` property. |
+| **Z-Axis** | The amount of space between probes along the z-axis. The space must be within the Box Shape component's dimension along the z-axis. | `0.0` to the Box Shape component's `Dimensions`-`Z` property. | The lowest value from the Box Shape component's `Dimensions`-`XYZ` property. |
```
[the remark this answers is not in the document]
Comment (chanmosq) [reply content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md:37]: After reviewing this, you're right the default value is 2.0 meters. I believe what I was observing may have been the probe spacing values updating when I update the mesh.

File: content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md [hunk 4]
```diff
@@ -5,18 +5,61 @@ description: "Open 3D Engine (O3DE) Diffuse Probe Grid component reference."
 toc: true
 ---
 
-{{< preview-new >}}
+The **Diffuse Probe Grid** component creates a volume of light probes that provide diffuse global illumination (GI) within the specified area. Each probe in the volume uses real-time ray tracing to capture the *irradiance*, or surrounding diffuse light environment. Real-time ray tracing casts several hundred rays in different directions around each probe. The probe stores lighting information at each point of intersection between the ray and the mesh. Then, it creates an irradiance texture out of the ray traced data and applies the texture to each mesh.
```
[the remark this answers is not in the document]
Comment (chanmosq) [reply content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md:8]: I think mesh may refer to the surrounding geometry? Or just the geometry within the volume. 

@dmcdiarmid-ly Can you specify?

[the remark this answers is not in the document]
Comment (chanmosq) [reply content/docs/user-guide/components/reference/atom/diffuse-probe-grid.md:8]: I believe the meshes that the rays intersect with refer to all of the geometry in the level that are in the direction of the ray. 

Hopefully this is more clear. I updated the sentence to: 
> At each point of intersection between the ray and the surrounding geometry, the probe stores lighting information.

**Review by willihay — APPROVED** [review]

**Review by micronAMZN — APPROVED** [review]

**Pull request merged**